test/parallel/test-worker-message-port-move.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable */
```

Comment:
Any special reason disable eslint for this file ?

Reply:
@gengjiawen That was because of the closure below which is actually re-compiled as a string… but thanks for the ping, it looks like adding `port` as a global is enough :) Done!